### DISCUSSION TITLE
Not received your education benefits after 4 days of my approval😢😢

### DISCUSSION THREAD

**Discussion by LISI0037** [community/community#138610] [GitHub Education]
### Hi everyone,

I have already finish the approval on 10 Sep, but the education benefits haven't received, please help me to solve this problem

**Comment by davevad93** [comment]
Hi @LISI0037 ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @LISI0037 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
